cmd/root.go

```diff
@@ -80,7 +82,7 @@ func init() {
 
 var (
 	createCmd = &cobra.Command{
-		Use:       "create <topology file>",
+		Use:       "create <topology file> [timeout sec]",
```

Comment:
timeout is a flag here, probably best to remove this part of the comment and rely on the auto documentation by cobra